src/components/MultiStepIndicator/components/StepIndicator/StepIndicator.jsx

```diff
@@ -40,13 +43,25 @@ const StepIndicator = ({
 
   const isStatusTransition = useCallback(() => prevStatusRef.current !== status, [prevStatusRef, status]);
 
+  const handleClick = () => {
+    if (onClick) onClick(stepNumber);
+  };
+
   // Event listeners for removing animation.
   useEventListener({
     eventName: "animationend",
     callback: disableStatusChangeAnimation,
     ref: componentRef
   });
 
+  const KEYS = [keyCodes.ENTER, keyCodes.SPACE];
```

Comment:
please move KEYS const outside the function as it will create new reference array each render cycle